src/http_seed_connection.cpp

```diff
@@ -279,10 +279,10 @@ namespace libtorrent {
 					// temporarily unavailable, retry later
 					t->retry_web_seed(this, retry_time);
 
-					std::string error_msg = to_string(m_parser.status_code()).data()
-						+ (" " + m_parser.message());
 					if (t->alerts().should_post<url_seed_alert>())
 					{
+						std::string const error_msg = to_string(m_parser.status_code()).data()
+							+ (" " + m_parser.message());
```

Comment:
interesting. the ``url_seed_alert`` should probably contain the actual ``error_code`` object too.